### PR TITLE
Newcomers_Guide.rst: Collect promotion requests in a form

### PR DESCRIPTION
**Pull request by adtac**

### CODE REVIEW

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -29,9 +29,9 @@ Request merged successfully, fill out our
 `survey form <http://coala.io/newform>`_. By doing so, you can help us make your
 experience better!
 
-Once you have achieved all these, just ask to be promoted on the chat
-(see step 1 below) and provide links to your reviews and merged Pull Requests.
-Then you'll be able to call yourself a coala developer!
+Once you've added completed all five tasks, please fill our
```
Comment (prnvdixit) [inline docs/Developers/Newcomers_Guide.rst]: "added" is relevant?

**Comment by prnvdixit** [comment]
In the doc, rather than asking for "What was the command line output when you ran coala?" - Shouldn't it be rephrased to "Give us a link to the command line output (through some code sharing platform (eg. pastebin)) when you ran coala"?

File: docs/Developers/Newcomers_Guide.rst (hunk 1 above)
Comment (adtac) [reply docs/Developers/Newcomers_Guide.rst]: Fixed :+1:

**Comment by adtac** [comment]
The form is easily fixable/editable later on :)

And yes, a pastebin link is preferred. I'll edit the form.

**Comment by prnvdixit** [comment]
Is there any particular utility (maybe for owners) of doing it this way rather than asking for promotion on the coala channel? (I had been mostly afk for quite some time - Is there any relevant discussion/issue thread?)

Also, that reminds me - Can this be merged without issue reference? (I just know about [this](https://github.com/coala/coala-bears/pull/2121#issuecomment-343418944)) :)

**Comment by adtac** [comment]
There's no issue reference for this commit. You could create a (fairly pointless) issue just for this PR and refer it, but I don't see the point. Reference links aren't necessary when there's no issue you're solving.

And yes, this helps us by making sure everyone completes all five tasks. Some people leave out something or the other, but by making all fields mandatory, they can't submit the form without a valid answer to each question.

**Comment by prnvdixit** [comment]
ack e3a3b9572f1fbf74ccfc71ea46c0def7e2efc114

**Comment by newbazz** [comment]
oh sorry i did not go through the convo.

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**